src/client/sandbox/node/element.js

```diff
@@ -161,9 +162,28 @@ export default class ElementSandbox extends SandboxBase {
         var overridedGetAttributeCore    = (el, attr, ns) => this._overridedGetAttributeCore(el, attr, ns);
         var overridedSetAttributeCore    = (el, attr, value, ns) => this._overridedSetAttributeCore(el, attr, value, ns);
         var overridedRemoveAttributeCore = (el, ns, arg) => this._overridedRemoveAttributeCore(el, ns, arg);
+        var overrideTextNode             = (node, parentNode) => {
+            if (!parentNode.tagName)
+                return;
+
+            var parentTagName = parentNode.tagName.toLowerCase();
+
+            if (parentTagName === 'script')
+                node.data = processScript(node.data, true, false);
+            else if (parentTagName === 'style')
+                node.data = processStyle(node.data, urlUtils.getProxyUrl);
+        };
+        var insertBeforeShadowUIRoot     = (bodyElement, child) => {
```

Comment:
And you will not need `bodyElement` argument, since it's will be `getRoot().parent`